.github/workflows/aarch64.yml

```diff
@@ -0,0 +1,51 @@
+name: Test aarch64
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - staging
+      - trying
+  schedule:
+    - cron: '0 0 * * 6'
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+        rust: [nightly]
+        include:
+          - os: macOS-latest
+            rust: 'nightly'
+            components: 'rust-src, llvm-tools-preview'
+            targets: 'aarch64-apple-darwin'
+          - os: ubuntu-latest
+            rust: 'nightly'
+            components: 'rust-src, llvm-tools-preview'
+            targets: 'aarch64-unknown-linux-gnu'
+
+    steps:
+    - uses: hecrj/setup-rust-action@v1
+      with: 
+         rust-version: ${{ matrix.rust }}
+         components: ${{ matrix.components || '' }}
+         targets: ${{ matrix.targets || '' }}
+    - uses: actions/checkout@v2
+      with:
+          lfs: true
+    - name: Check Cargo availability
+      run: cargo --version
+    - name: Install qemu (apt)
+      run: sudo apt-get update --fix-missing && sudo apt-get install qemu-system-aarch64
+      if: ${{ matrix.os == 'ubuntu-latest' }}
+    - name: Install qemu (macos)
+      run: |
+          brew install qemu
+      if: ${{ matrix.os == 'macOS-latest' }}
+    - name: Build loader (unix)
+      run: make arch=aarch64
```

Comment:
I think after building we should also do a step that runs the binary on qemu. 
`qemu-system-aarch64 -display none -smp 4 -m 1G -serial stdio -kernel target/aarch64-unknown-hermit-loader/debug/rusty-loader -machine raspi3`.
Since the rusty-hermit aarch64 implementation isn't quite ready yet, we'd also have to add a short timeout and `continue-on-error: true`.
We could then also check the stdout of the step to see if the loader successfully jumped into the hermit application (I could do this as a follow-up PR).

Reply:
Should I accept this PR? Or do you want to modify this PR?